library/src/main/java/com/google/android/exoplayer/extractor/webm/WebmExtractor.java

```diff
@@ -1437,6 +1461,33 @@ public void initializeOutput(ExtractorOutput output, int trackId, long durationU
       }
     }
 
+    public static final int WAVEFORMAT_SIZE = 18;
```

Comment:
All this stuff should be private rather than public (including the method). The constants should be defined at the top of the file (method at the bottom as it is now).